src/renderer/pages/Operations/components/FlexibleMultisigShell.tsx

```diff
@@ -0,0 +1,180 @@
+        </SmallTitleText>
+        <BodyText className="text-text-tertiary" align="center">
+          {t('operation.createFlexibleMultisig.description')}
+        </BodyText>
+
+        <div className="my-6 flex items-center">
+          {connection && isRejectAvailable && (
+            <RejectTxModal tx={tx} account={account} connection={extendedChain}>
+              <Button pallet="error" variant="fill">
+                {t('operation.rejectButton')}
+              </Button>
+            </RejectTxModal>
+          )}
+          {connection && (
+            <ApproveTxModal tx={tx} account={account} connection={extendedChain}>
+              <Button className="ml-auto">{t('operation.approveButton')}</Button>
+            </ApproveTxModal>
+          )}
+        </div>
+        {<Signatories signatories={tx.signatories} connection={extendedChain} events={events} />}
+
+        {/* TODO: add details */}
+      </Plate>
+    </div>
+  );
+};
+
+type SignatoriesParams = {
+  signatories: Signatory[];
+  connection: ExtendedChain;
+  events: MultisigEvent[];
+};
+type WalletSignatory = Signatory & {
+  wallet: Wallet;
+  status: SigningStatus | undefined;
+};
+
+const Signatories = ({ signatories, connection, events }: SignatoriesParams) => {
+  const { t } = useI18n();
+
+  const wallets = useUnit(walletModel.$wallets);
+  const contacts = useUnit(contactModel.$contacts);
+
+  const walletSignatories = signatories
+    .reduce<WalletSignatory[]>((acc, signatory) => {
+      const signatoryWallet = signatoryUtils.getSignatoryWallet(wallets, signatory.accountId);
+      const status = getSignatoryStatus(events, signatory.accountId);
+
+      if (signatoryWallet) {
+        acc.push({ ...signatory, wallet: signatoryWallet, status });
+      }
+
+      return acc;
+    }, [])
+    .sort((wallet) => (wallet.status === 'SIGNED' ? -1 : 1));
+
+  const walletSignatoriesIds = walletSignatories.map((a) => a.accountId);
+  const contactSignatories = signatories.filter((s) => !walletSignatoriesIds.includes(s.accountId));
+
+  return (
+    <Accordion>
+      <Accordion.Button buttonClass="px-2 py-1.5 mb-3">
+        <CaptionText className="uppercase text-text-secondary">
+          {t('operation.signatoriesTitleCount', { count: signatories.length })}
+        </CaptionText>
+      </Accordion.Button>
+      <Accordion.Content>
+        <div className="flex flex-col">
+          {walletSignatories.length > 0 && (
+            <ul className="flex flex-col gap-y-2">
+              {walletSignatories.map((signatory) => (
+                <SignatoryCard
+                  key={signatory.accountId}
+                  accountId={signatory.accountId}
+                  addressPrefix={connection.addressPrefix}
+                  status={signatory.status}
+                  explorers={connection.explorers}
+                >
+                  <div className="flex w-[100px] grow items-center gap-x-2 text-text-secondary">
```

Comment:
If this is possible 😉
```suggestion
                  <div className="flex w-25 grow items-center gap-x-2 text-text-secondary">
```